backend/models/user.js

```diff
@@ -0,0 +1,64 @@
+// const bcrypt = require("bcrypt"); Import when needed
```

Comment:
bcrypt hashing can be used while implementing authentication @doraemon127 
Check TSE Octavian for reference [link](https://github.com/TritonSE/TSE-Oktavian/blob/main/backend/models/user.js)

Reply:
I'll check this out, thanks!